JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -157,5 +157,19 @@
             var propertyValueDump: string = JSON.stringify(object);
             return objectTypeDump + propertyValueDump;
         }
+
+        /**
+         * Returns a boolean indicating whether the browser is IE or not.
+         * @param {string} userAgent - The user agent string for the browser
+         */
+        public static isBrowserIE(userAgent: string): boolean {
+            if (!userAgent) {
+                throw new Error('No user agent provided');
```

Comment:
As a corollary it's better practice to check for the features you need instead of looking for a specific browser. For example, if a future version IE supports the feature things will just work.